spec/support/shared_examples/contributor_resubscribes.rb

```diff
@@ -15,7 +15,7 @@
       expect(params[:organization_id]).to eq(organization.id)
       if adapter.eql?(WhatsAppAdapter::ThreeSixtyDialogOutbound::Text)
         expect(params[:payload][:to]).to eq(contributor.whats_app_phone_number.split('+').last)
-        expect(params[:payload][:template][:name]).to eq('welcome_message')
+        expect(params[:payload][:template][:name]).to eq("welcome_message_#{organization.project_name.parameterize.underscore}")
```

Comment:
```suggestion
        expect(params[:payload][:template][:name]).to eq("welcome_message_test_organization")
```

This assertion value should be hard-coded.

Reply:
I'm not sure about that @roschaefer ... Consider a test where the person setting it up wants to change the project name of the organization. Then it would not be `test_organization`, but something different.

Reply:
Well, then we obviously set the name of the organization per test-case, no?

Reply:
I think this is a stylistic preference that some devs on the team prefer and some don't, which indicates to me that we should allow space for differing styles, no?

Reply:
:laughing: You do you, I don't have a strong opinion.

I just had some bad experiences with assertion values that would change in an unexpected way. A classic is `expect(page).to have_text(expected_text)` and `expected_text` was `nil` which would always make the assertion true (-> false negative test).

In this case the risk is low. It's just a good routine.

It also reads a little better: `welcome_message_super_organization`

But do as you please, I won't argue over this :stuck_out_tongue:

Reply:
See 0caffe28
I generally agree with your reasons and I had a closer look, there is only one organization created that uses this shared example, so it was simple to make the change.